dns.tf

```diff
@@ -125,3 +125,15 @@ resource "azurerm_dns_cname_record" "release-ci-jenkins-io" {
 
   tags = local.default_tags
 }
+
+# A record for cert.ci.jenkins.io, accessible only via the private VPN
+# TODO: migrate this record to https://github.com/jenkins-infra/azure/blob/3aae66f0443c766301ae81f4d2aac5cec6032935/cert.ci.jenkins.io.tf#L14
```

Comment:
Why not doing that now?

Reply:
Because it is out of the scope of this PR as it requires WAY more work (importing existing cert.ci resources) and is out of scope